app/styles/user/main-user-view.sass

```diff
@@ -69,6 +69,51 @@
       padding: 5px 0px
       margin: 0px
       background: white
+.inventory
```

Comment:
Indent this so that it is scoped to be within the `#user-home` nesting.

Reply:
This should be fixed! 👍